requirements-test.txt

```diff
@@ -1,2 +1,3 @@
 coveralls
 openwisp-utils[qa]>=0.4.1
+mock==4.0.0
```

Comment:
are you sure we need to add this? I don't think we need. Mock is available in the standard library: https://docs.python.org/3/library/unittest.mock.html
Try using the mock library available in unittest please.